trestlebot/cli/commands/init.py

```diff
@@ -0,0 +1,130 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2024 Red Hat, Inc.
+
+""""
+Module for Trestle-bot init command
+"""
+import argparse
+import logging
+import sys
+from pathlib import Path
+
+import click
+from trestle.common.const import MODEL_DIR_LIST
+from trestle.core.commands.common.return_codes import CmdReturnCodes
+from trestle.core.commands.init import InitCmd
+
+from trestlebot.cli.config import make_config, write_to_file
+from trestlebot.cli.options.common import common_options
+from trestlebot.const import (
+    ERROR_EXIT_CODE,
+    TRESTLEBOT_CONFIG_DIR,
+    TRESTLEBOT_KEEP_FILE,
+)
+
+
+logger = logging.getLogger(__name__)
+
+
+def call_trestle_init(repo_path: Path, debug: bool) -> None:
+    """Call compliance-trestle to initialize workspace"""
+
+    verbose = 1 if debug else 0
+    trestle_args = argparse.Namespace(
+        verbose=verbose,
+        trestle_root=repo_path,
+        full=False,
+        govdocs=True,
+        local=False,
+    )
+    return_code = InitCmd()._run(trestle_args)
+    if return_code == CmdReturnCodes.SUCCESS.value:
+        logger.debug("Initialized trestle project successfully")
+    else:
+        logger.error(
+            f"Initialization failed.  Unexpted trestle error: {CmdReturnCodes(return_code).name}"
```

Comment:
Is "Unexpted" supposed to be unexpected? Not sure if it is an abbreviation for something else.

Reply:
> Is "Unexpted" supposed to be unexpected? Not sure if it is an abbreviation for something else.

Nope, just my bad spelling.  Ty!